### DISCUSSION TITLE
Pundit with Avo Actions

### DISCUSSION THREAD

**Discussion by oleksandr-storchak** [avo-hq/avo#1039] [General]
Hello guys! Please help me configure pundit policies with Avo Actions
I have some actions resources which work with another resource via Interactor and i will want to have policy for separate actions like :
def action_name?
end

**Comment by adrianthedev** [comment]
You could use [actions visibility](https://docs.avohq.io/2.0/actions.html#actions-visibility) and add your policy methods there. You could create a service that would forward action information to the policy.

**Reply by oleksandr-storchak** [reply]
Thank you!
